### PR TITLE
Translate assignment1 to assignment1-kore + trans.md

### PR DESCRIPTION
**Pull request by jiunbae**
+assignment1-kore.md
+assignment1-trans.md - for extra information while translating.

### CODE REVIEW

**Comment by gnujoow** [comment]
pull request 잘 보았습니다. 

**assigment1-trans.md**에 언급된 몇몇단어는 [glossary](http://aikorea.org/cs231n/glossary/)에 추가되어야 할 것같습니다.

이중 다음 단어는 이렇게 번역하는게 좋을 것 같습니다.
- SVM (그대로)
- validation data (검증 데이터)
- color histograms (색상 히스토그램)
- matplotlib (그대로 ,python 라이브러리)

나머지 단어들은 @myungsub 님이 봐주세요 ㅠ_ㅠ

---

개인적인 생각인데 추가적인 파일을 추가해서 번역을 해주시는것 보다 원본파일을 수정해서 pull request보내주시는것이 원본과 commit창에서 원본과 대조를 해볼 수 있기 때문에 리뷰하는데 좀 더 수월할 것같습니다.

**assignment1-trans.md**파일의 내용은 pull request를 보내주실 때 코멘트로 남겨주세요 merging할 때 해당파일도 같이 합쳐질 수 있을것 같습니다.

**Comment by jiunbae** [comment]
## 어휘 사전에 추가할만한 단어
- Softmax
- Implement
- Image Classification pipeline
- data-driven approach
- tuning
- hyperparameter
- proficiency
- raw pixels
- Histogram of Gradient (HOG) features
- insightful

## 이 부분은 이렇게 고쳤습니다.
- Setup에서 설명엔 local - virtual 순이지만 소 제목에서는 ###virtual - local 순 이기에 일치시켰습니다.

## 다른 문제될 사항
- Q6의 Or maybe you can experiment with a spin on the loss function 가 번역이 매끄럽게 되지 않아서 자문을 구합니다.

**Comment by myungsub** [comment]
@gnujoow 잠시 못보고 있던 사이에 답변달아주셔서 감사합니다.
어휘 사전에 추가할 만한 단어는 조금 논의해보고 오늘 중으로 정하도록 하겠습니다.
개인적인 의견으로는
- Softmax (영어 그대로)
- Implement : 구현하다 (ex. implement and apply a Softmax classifier --> Softmax 분류기 구현하고 적용해보기)
- Image Classification pipeline : 이미지 분류 파이프라인
- data-driven approach : (조금 애매합니다만..) 데이터기반 접근법(방법론)
- tuning : 튜닝
- hyperparameter : 초모수(?)
- develop proficiency in writing efficient vectorized code with numpy : numpy를 활용하여 효율적으로 벡터화된 코드를 작성하는데 익숙해지기
- raw pixels : 문맥상 적당히 풀어서 넘어가는 것이 좋을 것 같습니다. 예를 들어, '단순히 이미지 픽셀(화소)보다 더 고차원의 표현(representation)의 필요성~~~'
- Histogram of Gradient (HOG) features : 그라디언트 히스토그램 (이라고 처음에 명시한 이후부터는 그냥) HOG 특징
- insightful : 통찰

Q6 : Or maybe you can experiment with a spin on the loss function 
  --> 또는 손실 함수(loss function)을 조금씩 변형해가며 실험해볼 수도 있을 것입니다.
정도면 어떨까 싶습니다.

**Comment by jiunbae** [comment]
반영했습니다.

**Comment by jiunbae** [comment]
적절히 마무리되면 숙제의 자료들도 차례로 번역을 시작하겠습니다.

**Comment by myungsub** [comment]
Merge 한 뒤, hyperparameter만 다시 원래 영어로 바꾸었습니다. ('초모수'라는 단어가 너무 비호감이라는 사람들이 많아서..)
수고 많으셨습니다! 숙제 1에 포함된 파일들은 전부 번역해주실 건가요??

**Comment by jiunbae** [comment]
하나씩 시작하겠습니다.
자료가 좀 많으니 다른분들과 함께해도...

**Comment by myungsub** [comment]
어떤것부터 하실지 알려주시면 다른분과 일이 겹치지 않도록 역할분담표에 추가해 놓도록 하겠습니다.

**Comment by jiunbae** [comment]
features.ipynb
knn.ipynb
먼저 해보겠습니다.
ipynb는 보통 어떻게 열어서 편집하시나요? 저는 그냥 서브라임으로 열어서 쓰고있는데 더 좋은 방법이 있을까 해서.

**Comment by myungsub** [comment]
터미널에서 `jupyter notebook` 을 치고 브라우저에서 열어서 편집합니다.
필요한 dependency들이 있다면 설치해 주시면 되구요
원래 ipython notebook이었는데 jupyter로 업데이트됐어요
